### PR TITLE
fix: Savings and Loan account list do not update automatically

### LINKED ISSUE #1559: AccountsFragment - Savings and Loan account list do not update automatically.
**Summary:**

Changes made by the user in the savings or loan account list in the Accounts are not updated automatically instead the user has to swipe refresh to get the updated data, This might lead to a bad user experience.  

**Steps to reproduce:**

- Go to Accounts From Homepage

- Make changes to savings or loan account list

- Traverse back to accounts

**Expected behavior:**

Lists should be updated automatically when the user traverses back to accounts after making changes. 

**Observed behavior:**

The list is not updated automatically when the user traverses back to the accounts after making changes.

**Device and Android version:**

Xiaomi Redmi Note 4
Android 7.0
StockRom

**ScreenRecording:**

BugRun

### PR DESCRIPTION
**Pull request by ReCodee**
Fixes #1559 

Please Add Screenshots If there are any UI changes.

BugFix

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Review by rutvik-panchal — APPROVED** [review]

File: app/src/main/java/org/mifos/mobile/ui/fragments/AccountsFragment.kt [hunk 1]
```diff
@@ -207,6 +207,14 @@ class AccountsFragment : BaseFragment(), OnRefreshListener, AccountsView, Recycl
         currentFilterList = null
     }
 
+    /**
+     * This method is called when the fragment is visible to the user and actively running.
```
Comment (miPlodder) [inline app/src/main/java/org/mifos/mobile/ui/fragments/AccountsFragment.kt]: Let's remove this javadoc comment. Its the same as the parent class method which we are overriding

File: app/src/main/java/org/mifos/mobile/ui/fragments/AccountsFragment.kt [hunk 2]
```diff
@@ -207,6 +207,14 @@ class AccountsFragment : BaseFragment(), OnRefreshListener, AccountsView, Recycl
...
+     **/
+    override fun onResume() {
+        onRefresh()
+        super.onResume()
```
Comment (miPlodder) [inline app/src/main/java/org/mifos/mobile/ui/fragments/AccountsFragment.kt:211]: Is there any specific reason we are called `super.onResume` after `onRefresh` ? if no let's reverse the order of these calls

**Review by miPlodder — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/mifos/mobile/ui/fragments/AccountsFragment.kt (hunk 1 above)
Comment (ReCodee) [reply app/src/main/java/org/mifos/mobile/ui/fragments/AccountsFragment.kt]: Thanks for pointing this out, I've made the changes :)

File: app/src/main/java/org/mifos/mobile/ui/fragments/AccountsFragment.kt (hunk 2 above)
Comment (ReCodee) [reply app/src/main/java/org/mifos/mobile/ui/fragments/AccountsFragment.kt:211]: Done!

**Review by miPlodder — APPROVED** [review]

**Comment by miPlodder** [comment]
can you squash the commit msgs and follow this format `fix #issue-id : commit msg`

**Comment by ReCodee** [comment]
> can you squash the commit msgs and follow this format `fix #issue-id : commit msg`

Thanks, I've made the changes :)

**Comment by miPlodder** [comment]
Good job 👍

**Pull request merged**